### ISSUE TITLE
PhpDoc blocks note using prefix \ for all return types.

### ISSUE THREAD

**Issue by tomvlk** [Azure/azure-storage-php#23]
Currently the phpdoc blocks are not using the prefixed \ (root) before return class notation. This causes some IDE's to not show autocompletion (JetBrains).
## 

Sample fix in https://github.com/Azure/azure-storage-php/blob/master/src/Blob/BlobRestProxy.php#L715

``` php
    /**
     * Lists all of the containers in the given storage account.
     * 
     * @param Models\ListContainersOptions $options The optional parameters.
     * 
     * @return \MicrosoftAzure\Storage\Blob\Models\ListContainersResult
     *         ^ Adding the root prefix will fix autocompletion on some IDE's
     * @see http://msdn.microsoft.com/en-us/library/windowsazure/dd179352.aspx
     */
```
## 

I can change this myself and open pull if wanted to adopt the root() prefix.
## 

Even more issues are there when there is no full namespace defined in return, but relative. Some IDE's will not get the correct namespace path.

**Comment by katmsft** [comment]
I confirmed that this modification also enables Eclispe for PHP, which is our suggested IDE for the project, to perform auto completion. Pull request including the fix for this issue is welcome to be made.

**Comment by katmsft** [comment]
This issue is resolved by PR [#37](https://github.com/Azure/azure-storage-php/pull/37)
